### DISCUSSION TITLE
@urql/devtools (95kb) included in production bundle ?

### DISCUSSION THREAD

**Discussion by Biboswan** [urql-graphql/urql#1620] [Q&A] [answered]
```
import { devtoolsExchange } from '@urql/devtools';

const client = createClient({
  url: 'http://localhost:3001/graphql',
  exchanges: [devtoolsExchange, ...defaultExchanges],
});
```
What this does https://github.com/FormidableLabs/urql-devtools-exchange/blob/master/src/exchange.ts#L153
If I don't want devtoolsExchange  exchange in production, shall I explicitly do that through code 
if (!production) { exchanges = [devtoolsExchange, ...defaultExchanges] }

**Comment by JoviDeCroock** [comment, marked as answer]
When used in production `@urql/devtools` has a [400b impact](https://bundlephobia.com/result?p=@urql/devtools@2.0.3) on your bundle. 95kb would be the actual chrome plugin which never ships to your bundle but you install in your browser 😅
